api/v1/dspipeline_types.go

```diff
@@ -69,9 +69,13 @@ type APIServer struct {
 	// Include sample pipelines with the deployment of this DSP API Server. Default: true
 	// +kubebuilder:default:=false
 	// +kubebuilder:validation:Optional
-	EnableSamplePipeline bool   `json:"enableSamplePipeline"`
-	ArgoLauncherImage    string `json:"argoLauncherImage,omitempty"`
-	ArgoDriverImage      string `json:"argoDriverImage,omitempty"`
+	EnableSamplePipeline bool `json:"enableSamplePipeline"`
+	// Include ilab KFP pipeline with the deployment of this DSP API Server. Default: false
+	// +kubebuilder:default:=false
+	// +kubebuilder:validation:Optional
+	EnableIlabKFPPipeline bool   `json:"enableIlabKFPPipeline"`
```

Comment:
let's rename this (and all similar variables) to `EnableInstructlabPipeline`.  Its a has a little more clarity, and we should avoid using 'KFP' in DSP-specific functions like this one